nbs/tutorials/extensions.ipynb

```diff
@@ -0,0 +1,557 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #13.                # Otherwise...
</pre><p>only add comments where it's needed to explain why the code exists, or how it works. don't add comments that explain what the code <em>does</em> - assume the user knows how to read the code. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/nbdev/pull/1228/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
<p>Can you show an example of a real code cell and the before vs after the transformation that is applied by <code>LayoutProc?</code>  (This can also be used for a test if you like).  </p><p> </p><p>When you show this example, you can move some of the narrative that is currently in your comments into a markdown cell that is part of narrative that explains what <code>LayoutProc</code> is going to accomplish that goes into more detail than what the docstring provides.</p>

Reply:
<p>I think some of the comments that you have may be better suited to the narrative that you have below, basically describing what <code>LayoutProc</code> is going to do.   I think it is helpful to show the before vs. after of an entire code cell processed with <code>LayoutProc</code> so people can visually see the transformation that is happening a bit better.  </p><p> </p><p>Perhaps you want a margin comment on what <code>mk_cell</code> does or a link to it,  because I don't think most people will know what that is.  You may even consider hiding this cell as it could be distracting to the story. </p><p> </p><p> </p>

Reply:
<p>@hamelsmu this is the most critical part of the whole story imo, because it shows how to wrap it all together :) But let me rethink it's phrasing and presentation as you and Jeremy are right there's <em>much</em> more I can do with this! </p>